docs/cloud/azureai/generate-test-data-cloud.md

```diff
@@ -0,0 +1,38 @@
+# How to generate test data based on documents
+This guide will help you learn how to generate test data in a pipeline job on AzureML, so that you can integrate the created flow with existing pipelines and process a large amount of data.
+
+
+## Prerequisites
+
+1. Go through local test data generation [guide](../../how-to-guides/generate-test-data.md) and prepare your test data generation flow.
+2. Go to the [gen_test_data](../../../examples/gen_test_data) folder and run command `pip install -r requirements_cloud.txt` to prepare local environment.
+3. Prepare the environment required to run the component.
+    - Navigate to file [conda.yml](../../../examples/gen_test_data/conda.yml).
+    - For specific document file types, you will need to add extra packages in `conda.yml`:
+        > !Note: We use llama index `SimpleDirectoryReador` in this process. For the latest information on required packages, please check [here](https://docs.llamaindex.ai/en/stable/examples/data_connectors/simple_directory_reader.html).
+        - .docx - `docx2txt`
+        - .pdf - `pypdf`
+        - .ipynb - `nbconvert`
+
+4. Create cloud connection: [Create a connection](https://microsoft.github.io/promptflow/how-to-guides/manage-connections.html#create-a-connection)
```

Comment:
Cloud connection reference should be this https://microsoft.github.io/promptflow/cloud/azureai/quick-start.html#create-necessary-connections

Reply:
looks 4. should be after 5? Workspace creation should before connection creation